packages/cozy-harvest-lib/src/locales/fr.json

```diff
@@ -90,7 +90,8 @@
         "label": "Mise à jour :",
         "syncing": "En cours...",
         "unknown": "Indéterminée",
-        "format": "Le D MMMM YYYY [à] HH[:]mm"
+        "afterSomeMinutes": "Sync. il y a %{date}",
```

Comment:
nit: not sure if `date` is a correct name here (but I don't find any word that describe this)

Reply:
J'ai opté pour `times`